checkov/kubernetes/kubernetes_utils.py

```diff
@@ -209,3 +209,8 @@ def get_resource_id(resource: Dict[str, Any]) -> Optional[str]:
         labels.pop('__endline__', None)
         return f'{resource_type}.{namespace}.{str(labels)}'
     return None
+
+
+class K8sGraphFlags(TypedDict):
```

Comment:
```suggestion

@dataclass
class K8sGraphFlags:
```
just make it a `dataclass` 🙂  and don't forget to add the import and remove the now unneeded one (`from typing_extensions import TypedDict`)